qtc_packaging/debian_fremantle/control

```diff
@@ -1,4 +1,4 @@
-Source: xbmcremote
```

Comment:
I did rename, not drop it. :) Packaging/deployment works - tested on my N900. I swear. :)